package/MDAnalysis/coordinates/memory.py

```diff
@@ -278,10 +278,9 @@ def __init__(self, coordinate_array, order='fac',
                                  "array ({})"
                                  .format(provided_n_atoms, self.n_atoms))
 
+        self.dimensions = dimensions
```

Comment:
I'd store this as `_dimensions_array` to avoid someone accidentally using it thinking it was `ts.dimensions`?